app/views/web/teams/show.html.haml

```diff
@@ -0,0 +1,86 @@
+- title @team.title
+- content_for :meta do
+  %meta{ property: 'og:description', content: "#{@team.title} #МИЦ" }
+  %meta{ name:'description', content: "#{@team.title} #МИЦ" }
+.container.account
+  .row
+    .small-5.columns
+      - if @team.address
+        %iframe.team-map{ src: google_map_src(@team.address) }
+      - else
+        .team-card
+          .owner-head
+            = t('.owner')
+          .owner
+            = @owner.short_name
+            .locality= @owner.locality
+            .phone= @owner.mobile_phone
+            .email= @owner.email
+    .small-7.columns
+      .row
+        .small-11.push-1
+          .team-info
+            .team-title
+              = @team.title
+            - if @owner.email
+              .team-email
+                = @owner.email
+            .team-description
+              != @team.description
+.event-content.container
+  .participans-container
+    %h3.title
+      = t('.members')
+      %span#participants_count
+        = "(#{@members.count + 1})"
+    %ul.small-block-grid-5.participants
+      %li.owner.mic-member{ id: "participant_#{@owner.id}" }
+        = render_element 'mic_member', img_src: @owner.element_avatar, |
+                                       link: link_to_profile(@owner), |
+                                       name: @owner.short_name, |
+                                       role: t('.owner')
+      - @members.each do |member|
+        %li.participant.mic-member{ id: "participant_#{member.id}" }
+          = render_element 'mic_member', img_src: member.decorate.element_avatar, |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.<br>Line is too long. [83/80]